crates/wallet/src/lib.rs

```diff
@@ -4,13 +4,15 @@
 //!
 //! - `wallet_getCapabilities` based on [EIP-5792][eip-5792], with the only capability being
```

Comment:
i think `wallet_getCapabilities` also needs to be updated to new namespace or will this happen in a follow up PR?

Reply:
https://eips.ethereum.org/EIPS/eip-5792#wallet_getcapabilities it's part of eip so not target to be modified

Reply:
got it thanks for clarifying